client/src/components/CafeCard.tsx

```diff
@@ -22,9 +24,28 @@ const CafeCard = (props: CardProps) => {
     }
   }, [intersection?.isIntersecting, onIntersect]);
 
+  const handlePrevImage = () => {
+    setCurrentImageIndex((prevIndex) => (prevIndex === 0 ? cafe.images.urls.length - 1 : prevIndex - 1));
```

Comment:
```suggestion
    setCurrentImageIndex((prevIndex) => (prevIndex - 1 + cafe.images.urls.length) % cafe.images.urls.length);
```
분기로 인해 두 개의 expression이 생기는데요, 이렇게 하나로 만들 수도 있을 것 같네용
보시고 더 좋다 생각하는 코드로 채택해주세요~

Reply:
오홍...! 네네 좋습니닷